atlp-rwanda/e-comm-team-techsmith-fn
# #185218916 Login with Email and Password

## What does this PR do?
This PR consists of a login a simple and fantastic login page
## Description of Task to be completed?
This task adds a feature of allowing users to be able to interact with a very user friendly login page which they will help in the signing to their account
## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout ft-login-page-185218916`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm run start`

- Check the application's documentation on Storybook: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?
[#185218916](https://www.pivotaltracker.com/story/show/185218916)
## Screenshots (if appropriate)
**Laptop view**
![Screen Shot 2015-09-21 at 11 11 17 AM](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/108126328/eeaea489-e9b8-46b3-b984-d5f236c04b11)
**Validation view**
![Screen Shot 2015-09-25 at 8 02 24 AM](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/108126328/7665163a-0240-427f-9899-7d59b0b1d1e1)


**Mobile phone view**
![Screen Shot 2015-09-21 at 11 15 34 AM](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/108126328/55f90727-cdb0-4069-82ad-ceb7c44f6b61)
